flask_resty/spec/__init__.py

```diff
@@ -0,0 +1,7 @@
+"""Flask-Resty plugin for apispec. Allows passing a Flask-Resty
+`view` to `APISpec.add_path <apispec.APISpec.add_path>`
+"""
+
+from .plugin import setup, schema_path_helper  # noqa
+from .swagger_utils import *  # noqa
```

Comment:
Also, can this just be `utils`? And the below just `declaration`? You're already making a separate package; no need to further namespace.